sdk/modelsrepository/Azure.Iot.ModelsRepository/src/RepositoryHandler.cs

```diff
@@ -16,19 +17,22 @@ internal class RepositoryHandler
         private readonly IModelFetcher _modelFetcher;
         private readonly Guid _clientId;
         private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly Uri _repositoryUri;
+        private readonly ResolverClientOptions _clientOptions;
 
-        public Uri RepositoryUri { get; }
```

Comment:
The only other place that these properties are accessed is in the ResolverClient that actually passes these down to the RepositoryHandler itself, no need to have backward references as ResolverClient knows about these values itself.